.NET/Microsoft.Recognizers.Definitions.Common/English/NumbersDefinitions.cs

```diff
@@ -152,6 +152,8 @@ public static class NumbersDefinitions
 		public static readonly Dictionary<string, long> OrdinalNumberMap = new Dictionary<string, long>
 		{
 			{ @"first", 1 },
+			{ @"first one", 1 },

```

Comment:
Can't we do some normalization before checking this map? Instead of adding a lot of redundant values.